src/app/containers/LinkedData/helpers.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable import/prefer-default-export */
+export const cleanLinkedData = linkedData => {
```

Comment:
Alternatively, could we just have a utilities folder, and have `cleanLinkedData` as a separate file + test?

Reply:
Seems like an overly complex way to conditionally add 2 properties

Reply:
inside of https://github.com/bbc/simorgh/pull/6901/files#diff-0e25abbcfed44609b2085dac70092114R62

can you not just do?

```js
  const linkedData = {
    '@type': type,
    url: canonicalNonUkLink,
    image,
    mainEntityOfPage,
...(type === 'WebPage' && { publisher, thumbnailUrl: defaultImage }),
```

Reply:
I might be guilty here of over-optimizing, but it seems to be that there could be specific changes down the line. I guess I could go ahead and include it in the main file

Reply:
Thanks for the changes. We could consider something like that in the future if we need it though